src/main/java/com/whytowait/core/annotations/enums/RequestSource.java

```diff
@@ -0,0 +1,16 @@
+package com.whytowait.core.annotations.enums;
+
+import lombok.Getter;
+
+@Getter
+public enum RequestSource {
+    HEADER("merchant-id"),
+    PARAMS("merchantId"),
+    BODY("merchantId");
+
+    private final String source;
```

Comment:
remove source, add source to annotation definition

Reply:
Source is added to annotation defination only, this is used for getting the field value.